### ISSUE TITLE
Cut 1.15.0-beta.2 release

### ISSUE THREAD

**Issue by Bubblemelon** [kubernetes/sig-release#654]
Scheduled to happen: Wed, 2019-06-5

- [x] screen shot master's unhealthy testgrid boards and add them as a comment 
  

- [x] stage & release
  - [x] notify [#sig-release](https://kubernetes.slack.com/messages/C2C40FMNF/): [see notification](/https://kubernetes.slack.com/archives/C2C40FMNF/p1559792315015600)
  
  - [x] collect metrics, links, ... and add them as a comment 
  

- [x] send notification [email](https://groups.google.com/forum/#!topic/kubernetes-announce/5BxZW1NdplI)

/milestone v1.15
/sig release  
/area release-team  

/assign @Bubblemelon

**Comment by Bubblemelon** [comment]
### Testgrid dashboards 
Failling boards:
- [sig-release-master-informing](https://testgrid.k8s.io/sig-release-master-informing)
- [sig-release-1.15-all](https://testgrid.k8s.io/sig-release-1.15-all)

<details><summary>sig-release-master-informing</summary><p>

![image](https://user-images.githubusercontent.com/12985181/59004612-1281fe80-87cf-11e9-9bdc-4e4fedb9a5ca.png)
</p></details>

<details><summary>sig-release-1.15-all</summary><p>

![image](https://user-images.githubusercontent.com/12985181/59004686-5f65d500-87cf-11e9-8621-4b3f942fd964.png)

![image](https://user-images.githubusercontent.com/12985181/59004733-8de3b000-87cf-11e9-973e-b2d398680987.png)


</p></details>



Bottom line, these tests have a lot of failed builds.

**Comment by Bubblemelon** [comment]
v1.15.0-beta.2 | June 5 |  |  
-- | -- | -- | -- |
step | command | link | duration
mock stage | - |   |   |   |  
mock release | - |   |   |   |  
stage | ./gcbmgr stage release-1.15 --build-at-head --nomock | https://console.cloud.google.com/gcr/builds/e1da2c34-651a-45a7-b628-614674fcae0b?project=648026197307 | 52m
release | ./gcbmgr release --nomock release-1.15 --buildversion=v1.15.0-beta.1.446+bfa37555228b66 | https://console.cloud.google.com/gcr/builds/3bad7e15-d449-40a1-82a0-08056833be9e?project=648026197307 | 18m
